### DISCUSSION TITLE
asset.read

### DISCUSSION THREAD

**Discussion by ghost** [nextcord/nextcord#274] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/94598301/143471805-46350677-ad63-4dae-8172-a5a507cbf606.png)

asset.read seems to be missing, help pls!

**Comment by ooliver1** [comment, marked as answer]
Try reading the error, "asset" is a string not an asset, is asset a url or something

**Reply by ghost** [reply]
im doing a wanted command and suddenly shows me it

**Reply by ghost** [reply]
heres the commands do i have anything to be fix?
![image](https://user-images.githubusercontent.com/94598301/143473690-be98768e-4434-4011-a1d8-88fb2d42a266.png)

**Reply by ooliver1** [reply]
You can do
```py
formatted = codeblocks
```
Btw, but you're getting the url, without .url is the asset
